org.eclipse.scanning.event/src/org/eclipse/scanning/event/remote/_Positioner.java

```diff
@@ -48,7 +49,9 @@ public void disconnect() throws EventException {
 	
 	public void init()  throws EventException {
 		requester = eservice.createRequestor(uri, EventConstants.POSITIONER_REQUEST_TOPIC, EventConstants.POSITIONER_RESPONSE_TOPIC);
-	    requester.setTimeout(RemoteServiceFactory.getTime(), RemoteServiceFactory.getTimeUnit()); // Useful for debugging testing 
+		long timeout = Long.getLong("org.eclipse.scanning.event.remote.positionerTimeout", 100); 

```

Comment:
Maybe make the default timeout a defined constant at the top of the class (here & elsewhere)